### PR TITLE
UPDATE: Add forwarding ref to inputs

### PR DESCRIPTION
**Pull request by wEprous**
<br/><br/><br/><url>LiveURL: https://orbit-components-add-forwarding-ref.surge.sh</url>

### CODE REVIEW

File: src/InputField/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Table below contains all types of the props available in InputField component.
 | help          | `React.Node`      |              | The help to display beneath the InputField.
 | label         | `string`          |              | The label for the InputField. [See Functional specs](#functional-specs)
 | inlineLabel   | `boolean`         |              | If true the label renders on the left side of input
+| inputRef      | `func`            |              | Prop for forwarding ref the of input. [See Functional specs](#functional-specs)
```
Comment (tomashapl) [inline src/InputField/README.md]: should be just `ref` everybody know this prop in react

File: src/InputField/README.md [hunk 2]
```diff
@@ -50,3 +51,20 @@ Table below contains all types of the props available in InputField component.
...
 * You can use `string` for currency InputField, or `React.Node` for InputField with icon.
+
+* `inputRef` can be used for example auto-focus the elements immediately after render.
+```jsx
+class Component extends React.PureComponent<Props> {
+  componentDidMount() {
+    this.inputRef.current.focus();
+  }
+
+  inputRef = React.createRef();
```
Comment (tomashapl) [inline src/InputField/README.md]: Should be in constructor

```
constructor(props) {
    super(props)
    this.inputRef = React.createRef();
}
```

File: src/InputGroup/index.js [hunk 3]
```diff
@@ -138,10 +138,6 @@ class InputGroup extends React.PureComponent<Props, State> {
     this.isFilled();
   }
 
-  componentDidUpdate() {
-    this.isFilled();
```
Comment (tomashapl) [inline src/InputGroup/index.js]: Why this removal ?

Comment (wEprous) [reply src/InputGroup/index.js]: Tests were failing, I just deleted and forgot about it.
With upgrading enzyme to the new version, tests will fail for this component. `componentDidUpdate` calling itself in an infinite loop, we need to fit it in separate PR.

Comment (wEprous) [reply src/InputGroup/index.js]:
```
RangeError: Maximum call stack size exceeded
      144 | 
      145 |   isFilled = () =>
   > 146 |     this.setState({
             |          ^
      147 |       filled: !React.Children.map(
      148 |         this.props.children,
      149 |         child => child.props.value !== undefined && child.props.value !== "",
```

**Comment by wEprous** [comment]
Should be gone @tomashapl . We just need to fix the InputGroup :)

File: src/InputField/README.md (hunk 1 above)
Comment (tomashapl) [reply src/InputField/README.md]: @wEprous why inputRef ?

Comment (wEprous) [reply src/InputField/README.md]: It's outdated. Already changed 🙂

**Comment by wEprous** [comment]
Rebased and resolved conflicts 🙂

**Comment by tomashapl** [comment]
@wEprous Great! Thanks 👏

**Pull request merged**